src/oemof_tabular_plugins/datapackage/post_processing.py

```diff
@@ -119,6 +119,9 @@ def infer_busses_carrier(energy_system):
                 if hasattr(node, attribute):
 
                     bus_label = getattr(node, attribute).label
+                    print(
```

Comment:
on the line 118 where you have  `for attribute in ("bus", "from_bus"):`

you can simply add the column name you want to read for inferring carrier `for attribute in ("bus", "from_bus", "from_bus_0", "from_bus_1"):`

Reply:
it's actually more complicated than this because the MIMO now there are problems with having one carrier but multiple busses associated with it. Should I write a different infer_busses_carrier method for MIMO?

![image](https://github.com/rl-institut/oemof-tabular-plugins/assets/53213814/6afa6eb5-a8d8-4664-b3d1-f3da74e45c90)

Reply:
It is the reverse, the error says one bus has two carriers, which does not make sense indeed

Reply:
yes true, but because of the MIMO only having one carrier and two busses (so 'water' being the carrier as this is the primary), this leads to this error with the ac-elec-bus having these two carriers attached to it